src/sentry/static/sentry/app/components/group/externalIssueActions.jsx

```diff
@@ -260,6 +315,33 @@ export class SentryAppExternalIssueActions extends React.Component {
   }
 }
 
+const Wrapped = withApi(SentryAppExternalIssueActions);
```

Comment:
We generally export this as default and then export the undecorated component with named exports

Reply:
The `default` in this file is actually another class. That one, and this class, are essentially the same with slight differences (one is to render Integration things, while this one is for Sentry App things), hence keeping them in the same file.

Ultimately I'd like to consolidate this all into a single component, but that requires coercing Integrations and SentryApp objects into something with a consistent interface. I went down that path briefly, but it's a lot bigger undertaking than we have the resources for right now.

I can move these classes out into their own files, though, if we'd prefer that (despite the similarities).

Reply:
Yeah I think it's preferable to have this in its own file (We try to stick to 1 exported component per file)